### PR TITLE
Enable display scaling on Android tablets

### PR DESCRIPTION
**Pull request by bvschaik**
@crudelios Could you perhaps check if this breaks anything on Vita/Switch?

### CODE REVIEW

File: src/platform/screen.c [hunk 1]
```diff
@@ -230,14 +230,33 @@ int system_scale_display(int display_scale_percentage)
...
+#ifndef __ANDROID__
+    if (system_is_fullscreen_only()) {
+        return 0;
+    }
+#endif
     int width, height;
     SDL_GetWindowSize(SDL.window, &width, &height);
-    return get_max_scale_percentage(width, height);
+#ifdef __ANDROID__
+    int maxOrientation1 = get_max_scale_percentage(width, height);
```
Comment (crudelios) [inline src/platform/screen.c:246]:
```suggestion
    int max_orientation_portrait = get_max_scale_percentage(width, height);
```

File: src/platform/screen.c [hunk 2]
```diff
@@ -230,14 +230,33 @@ int system_scale_display(int display_scale_percentage)
...
+    int maxOrientation2 = get_max_scale_percentage(height, width);
```
Comment (crudelios) [inline src/platform/screen.c:247]:
```suggestion
    int max_orientation_landscape = get_max_scale_percentage(height, width);
```

**Review by crudelios — COMMENTED** [review]
It should work but I'll check,

Anyway, here's a small oversight I noticed.

File: src/platform/screen.c (hunk 1 above)
Comment (bvschaik) [reply src/platform/screen.c:246]: Since we don't know whether the current orientation is portrait or landscape I opted for simply "1" and "2".

Comment (crudelios) [reply src/platform/screen.c:246]: Makes sense. What about `max_orientation_current` and `max_orientation_alternative` though?

Comment (bvschaik) [reply src/platform/screen.c]: Good suggestion. I opted for `max_scale_current_orientation` and `max_scale_alternative_orientation` to make it clear we're dealing with a maximum scale.

**Pull request merged**